app/controllers/spree/admin/general_settings_controller.rb

```diff
@@ -19,6 +20,18 @@ def update
 
         redirect_to edit_admin_general_settings_path
       end
+
+      private
+
+      def combine_available_units_params
```

Comment:
maybe merge would be a better name than combine here? I didnt understand this initial. Now I think it's merging params :-)